motor_control/throttle/Button.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BUTTON_THROTTLE
+#define BUTTON_THROTTLE
+
+#include "base_throttle.h"
+
+class Button: public Throttle{
+    public:
+    Button(int ft);
+    virtual int readThrottle();
```

Comment:
It looks like the implementation for this should go under `compute_motor_value()`. There is also no need for the `virtual` keyword here.